_summaries/Rae2019compressive_1.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: summary
+title: Summary
+paper: Rae2019compressive_1
+# Please fill out info below
+author: zayne-sprague
+score: 9
+---
+
+## What's the big idea
+
+- By compressing memories, transformers are able to model long range sequences more efficiently and effectively, 
+outperforming state-of-the-art benchmarks on various sequence based tasks.
+  
+## What's Wrong With Attention
+
+- Attention is expensive in both computation time and memory
+  
+- It's also not a great model of how memory really works (most do not remember every detail of every second of their lives, yet that is the transformer model)
+
+- Current architecture is forced to move on and limit themselves to a fixed size window of attention
+
+![animation of transformerXL](rae2019compressive_1_custom_b.gif)
+
```

Comment:
Perhaps you can mention that TransformerXL has been proposed as a way to extend the memory window size, but it simply discards long-range memory.